examples/snap.js

```diff
@@ -73,7 +73,7 @@ Modify.init();
 
 var optionsForm = document.getElementById('options-form');
 
-var Draw = {
+var ExampleDraw = {
```

Comment:
Curious if the linter caught this after rename, if you searched for 'var Draw' before, or if something else.

Reply:
The linter warns, as the variable was already used.